src/components/commons/CommonBookCard.tsx

```diff
@@ -0,0 +1,132 @@
+import {
+  Card,
+  CardMedia,
+  CardContent,
+  Typography,
+  Rating,
+  Box,
+  SxProps,
+  Stack,
+} from '@mui/material';
+import PeopleIcon from '@mui/icons-material/People'; // 사람 아이콘 추가
+
+interface CommonBookCardProps {
+  image: string; // 이미지
+  title?: string; // 제목
+  author?: string; // 저자
+  customerReviewRank?: number; // 0~10점 값
+  priceStandard?: number; // 정가 가격
+  ratingCount?: number; // 평가 인원 수
+  onClick?: () => void; // 상세페이지 이동 을 위한 onClick 함수 생성
+  sx?: SxProps; // MUI의 sx 스타일 속성
+}
+
+const CommonBookCard = ({
+  image,
+  title,
+  author,
+  customerReviewRank,
+  priceStandard,
+  ratingCount,
+  sx,
+  onClick,
+}: CommonBookCardProps): JSX.Element => {
+  const convertedRating = customerReviewRank ? customerReviewRank / 2 : 0; // 10점 만점 -> 5점 만점으로 변환
+  const displayRating = Number.isInteger(convertedRating)
+    ? convertedRating.toFixed(0) // 정수일 경우 소수점 제거
+    : convertedRating.toFixed(1); // 소수점 1자리 유지
+
+  return (
+    <Card
+      sx={{
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        cursor: 'pointer',
+        ...sx,
+      }}
+      onClick={onClick}
+    >
+      <CardMedia
+        component="img"
+        image={image}
+        alt={title || '책 이미지'}
+        sx={{
+          height: 200, // 고정된 높이
+          objectFit: 'fill', // 이미지를 cover 방식으로 표시
```

Comment:
```suggestion
          height: 160, // 고정된 높이
          objectFit: 'contain', // 이미지를 contain 방식으로 표시
```

이것도 디자인 한 번 봐주세용! 만약에 반영 안 하더라도 주석은 수정해야 할 것 같습니다! fill인데, cover로 되어 있어요!

<img width="872" alt="image" src="https://github.com/user-attachments/assets/13fbd89e-f8cc-441d-a896-766e888190b1" />

Reply:
contain이 더 디자인적으로 괜찮은것 같네용